tests/test_response.py

```diff
@@ -28,3 +36,8 @@ def test_it_checks_for_another_template_on_current_version(self, response):
     def test_it_sets_connection_on_init(self, response):
         assert not hasattr(RedisTemplateResponse, 'connection')
         assert response.connection
+
+    def test_it_pings_connection_on_init(self):
+        response = self.response()
```

Comment:
I don't think it is much different, the test itself changed so I can just include it as an arg to the test method as well.